fortinet_fortigate/icon_fortinet_fortigate/util/util.py

```diff
@@ -28,7 +28,9 @@ class Helpers(object):
                        " The offended source is temporarily blocked for certain amount of time.",
               "assistance": "Wait out the access timeout."},
         500: {"cause": "Internal Server Error: Internal error when processing the request.",
-              "assistance": "Contact support for assistance."}
+              "assistance": "The most common causes of this error are invalid requests."
+                            "Check that the address has not already been created."
```

Comment:
```suggestion
                            "This can happen when an object you're trying to create already exists or when an object you're trying to remove doesn't exist."
```